RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -90,17 +85,13 @@ metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
       double jpt  = jet->pt();
       double jeta = jet->eta();
       double feta = std::abs(jeta);
-      double c = std::cos(jet->phi());
-      double s = std::sin(jet->phi());
+      double c = jet->px()/jet->pt();
```

Comment:
chances are that the compiler optimizes away repeated calls, but it's unclear if it does it in this particular case (several nested operator calls).